docs/modeling/physical_models.rst

```diff
@@ -288,3 +288,38 @@ The maximum circular velocity and radius of maximum circular velocity are availa
     # Display plot
     plt.tight_layout(rect=[0, 0.03, 1, 0.95])
     plt.show()
+
+
+.. _Cosmologies:
+
+Cosmologies
+===========
+
+The instances of the |Cosmology| class (and subclasses) include
+|Cosmology.to_format|, a method to convert a Cosmology to another python
+object. Relevant here is the conversion of any redshift method to a
+:class:`~astropy.modeling.FittableModel` instance, using the argument
+``format="astropy.model"``.
+Each |Cosmology| :class:`~astropy.cosmology.Parameter` is converted to a
+:class:`astropy.modeling.Model` :class:`~astropy.modeling.Parameter`
+and the redshift-method to the model's ``__call__`` / ``evaluate``.
+Now you can fit cosmologies with data!
```

Comment:
```suggestion
object. Specifically, any redshift method can be converted to a
:class:`~astropy.modeling.FittableModel` instance using the argument
``format="astropy.model"``.
During the conversion, each |Cosmology| :class:`~astropy.cosmology.Parameter` is converted to a
:class:`astropy.modeling.Model` :class:`~astropy.modeling.Parameter`, while the redshift-method becomes the model's ``__call__`` / ``evaluate`` method.
This means cosmologies can now be fit with data!
```

The word choice made it difficult for me to follow on the first read. Above is my suggested reword.

Reply:
Thanks! I use this, but adjust the line lengths to match the rest of the document.